modules/auxiliary/gather/search_email_collector.rb

```diff
@@ -62,7 +61,6 @@ def search_yahoo(targetdom)
     print_status("Searching Yahoo for email addresses from #{targetdom}")
     response = ""
     emails = []
-    header = { 'User-Agent' => "Mozilla/5.0 (Windows; U; Windows NT 5.1; en-US) AppleWebKit/525.13 (KHTML, like Gecko) Chrome/4.0.221.6 Safari/525.13"}
```

Comment:
Do we not want to replace these with a user-agent generated via `Rex::UserAgent`?

Reply:
Right, this one uses the built-in Ruby `Net::HTTP`, so does need it explicitly specified.

Reply:
Okay well I officially hate GitHub's views but apparently I had it on the wrong commit or something cause if I view all commits it shows this code is right but if I view certain commits it had these changes wrong. Lolz oh well. 

Anyway this code looks good now, marking this as resolved; as noted if we were to default to `Net::HTTP` we would always get a user agent of `Ruby` from the code at https://github.com/ruby/net-http/blob/ca7bb8c51c6f04555d0b5173556154bd3483777e/lib/net/http/generic_request.rb#L48 which is undesireable.